src/components/sdk/MeetingControls/AudioInputVFControl.tsx

```diff
@@ -119,6 +119,7 @@ const AudioInputVFControl: React.FC<Props> = ({
     addVoiceFocus,
     device,
     devices.length,
+    devices[0].label,
```

Comment:
hmm how does this change actually make it so that VF is enabled? Won't this change just render the dropdown again with updates?

Reply:
isn't label in this case still `default`?

Reply:
No, device id is `default` - the label would change from `Macbook Microphone` to `Airpods`

Reply:
The `devices[0].deviceId` is still `default`, but `devices[0].label` is changed from `Internal Microphone` to `AirPods`

Reply:
> hmm how does this change actually make it so that VF is enabled? Won't this change just render the dropdown again with updates?

Yep, exactly. The current device is actually set to the new device, but the change does not reflect on the dropdown list. We only need to update the dropdown list

Reply:
We should handle this using the audioInputsChanged event from JS SDK. This looks a bit hacky.

Reply:
Are you sure VF would be enabled still? In AudioInputProvider,  we call 
`await audioVideo?.chooseAudioInputDevice(selectedInputRef.current);` - This is called directly (not via meetingManager) - how would the selectedAudioInputTransformDevice get updated?

Reply:
>We should handle this using the audioInputsChanged event from JS SDK. This looks a bit hacky.  

Actually `audioInputsChanged` has been trigged twice as there are 2 steps for AirPods to be set as default device and in the second time, only `audioInputs` changes. This dropdown list is only updated when length of `audioInputs` or `audioInputs[0].deviceId` changes and will stay the same when `audioInputs[0].label` changes. 

Since we can not put `audioInputs` as a dependency, so use `audioInputs[0].label` instead.

Reply:
> Are you sure VF would be enabled still? In AudioInputProvider, we call
> `await audioVideo?.chooseAudioInputDevice(selectedInputRef.current);` - This is called directly (not via meetingManager) - how would the selectedAudioInputTransformDevice get updated?

Without VF, when this `audioVideo?.chooseAudioInputDevice` is called, the `selectedAudioInputTransformDevice` gets updated

Reply:
Shouldn't we just remove devices.length and devices[0].label and just use devices here?

Reply:
I'd love to...but if we use `devices` here would cause `Maximum update depth exceeded` issue

Reply:
> Are you sure VF would be enabled still? In AudioInputProvider, we call
> `await audioVideo?.chooseAudioInputDevice(selectedInputRef.current);` - This is called directly (not via meetingManager) - how would the selectedAudioInputTransformDevice get updated?

Just confirmed, our `selectedAudioInputTransformDevice` won't change, it would keep as string `default`. But the `default` device changes in `audioInputs`, no matter whether VF is enabled or not.

Reply:
So the second time the audioInputsChanged is called, the device list is technically the same but the 'default' device has a different label, which is why we have to change based on that label. Makes sense

Reply:
Yep, exactly